modules/core/types/codec.go

```diff
@@ -17,6 +16,5 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	connectiontypes.RegisterInterfaces(registry)
 	channeltypes.RegisterInterfaces(registry)
 	solomachine.RegisterInterfaces(registry)
-	ibctm.RegisterInterfaces(registry)
```

Comment:
core IBC will no longer include clients by default. This is a feature in my opinion, as it makes core IBC neutral to which light clients should be registered. This is aligned with the IBC app registration approach which requires chain developers to add code to register an application. Luckily adding a client is extremely trivial